### PR TITLE
[MRG, DOC] consolidate installation instructions, add conda

### LINKED ISSUE #206: Now available from conda-forge
Hello, I've packages `autoreject` for `conda-forge`:
https://github.com/conda-forge/autoreject-feedstock

A bot will automatically discover new releases on PyPI and update the package accordingly.

I wonder if we should include this in the installation instructions?

### PR DESCRIPTION
**Pull request by sappelhoff**
closes #206 

Instead of maintaining the same information twice (once in index.rst, once in README.rst). I suggest to only maintain it in one place and include that information in the other page,

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -40,18 +68,52 @@ These are the dependencies to use ``autoreject``:
...
+We also implement RANSAC from the PREP pipeline.
+The API is the same:
+
+.. code:: python
+
+	>>> from autoreject import Ransac
+	>>> rsc = Ransac()
+	>>> epochs_clean = rsc.fit_transform(epochs)  # doctest: +SKIP
+
+For more details check out the :ref:`example to automatically detect and repair bad epochs <sphx_glr_auto_examples_plot_auto_repair.py>`.
```
Comment (jasmainak) [inline README.rst:104]: This seems broken

**Comment by jasmainak** [comment]
LGTM other than minor nitpick. Thanks @sappelhoff !

File: README.rst (hunk 1 above)
Comment (sappelhoff) [reply README.rst:104]: Yes, this will not work in the GitHub rendering, only in the actual docs.

Should I remove the link? Do you have another idea?

Comment (jasmainak) [reply README.rst:104]: You can link with the URL for the dev version. I think that should be good enough

**Comment by sappelhoff** [comment]
CI breaks at downloading MNE sample data. It's kind of flaky, because it passed just before ... same thing is currently also broken in main.

Nothing to fix in this PR, I think. Ready to merge :-)

**Comment by jasmainak** [comment]
Thanks @sappelhoff !

**Pull request merged**